### ISSUE TITLE
WriteComment should ignore empty ones

### ISSUE THREAD

**Issue by emicklei** [magiconair/properties#3]
If no comment was set for a key,value pair then I expect the WriteComment to not write the extra lines.
```
#
eddy.aq.concurrent.consumers = 10

#
eddy.aq.max.concurrent.consumers = 10
```

vs

```
eddy.aq.concurrent.consumers = 10
eddy.aq.max.concurrent.consumers = 10
```

**Comment by magiconair** [comment]
I've pushed the issue_3 branch which should fix that. Pls have a look.

**Comment by emicklei** [comment]
thanks, will have a look tomorrow

On Wed, Apr 8, 2015 at 5:36 PM, Frank Schröder <notifications@github.com>
wrote:

> I've pushed the issue_3 branch which should fix that. Pls have a look.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/magiconair/properties/issues/3#issuecomment-90953241>.
>



-- 
^ Ernest Micklei

http://ernestmicklei.com

**Comment by magiconair** [comment]
Fixed in v1.5.2
